uint8 id
string data_type
float64 value
string status